charts/kubernetes-external-secrets/templates/deployment.yaml

```
@@ -27,6 +27,10 @@ spec:
       {{- end }}
     spec:
       serviceAccountName: {{ template "kubernetes-external-secrets.serviceAccountName" . }}
+      {{- if .Values.image.imagePullSecrets }}
+      imagePullSecrets:
+        - name: {{ .Values.image.imagePullSecrets }}
```

```suggestion
        {{- toYaml .Values.imagePullSecrets | nindent 8 }}
```

not sure if the 8 is correct, please validate 😅